### ISSUE TITLE
Batch gradient descent not implemented

### ISSUE THREAD

**Issue by margheritatonon** [margheritatonon/pytorch-image-rotation#1]
`batch_size` is a hyperparameter that can be tuned in the YAML configuration files, but does not appear in the training step. Need to update train.py to take batch size into account. Transformer model is very slow to train.
